README.md

```diff
@@ -36,6 +36,10 @@ note: z means sapling shielded addresses.
 
 Contributions are very much welcomed! Please read our [Contributing Guidelines](/CONTRIBUTING.md) and [Code of Conduct](/CONDUCT.md). Our backlog has many Issues tagged with the `good first issue` label. Please fork the repo and make a pull request for us to review.
 
+# Project Requirements
```

Comment:
Shall there be a section in the documentation that actually explains (or links to an explanation) how to properly use this tool in the context of this project?

Reply:
Technically you don't have to do anything to use this tool, it's done for you in the build phases, if you have the tool installed.

Reply:
How should a contributor use it in the case it's needed to add or modify resources?

I haven't used SwiftGen so I could use myself as an Example. If I were to change something that needs additional resources, you name it. What's the proper approach to do so in this project?

Reply:
A contributor only needs to work with the assets in the same way they normally would, there are no extra steps involved.  If you would like to add a font or a color for example, using the existing Fonts folder or the existing Colors asset file will result in SwiftGen giving you updated helpers in the generated file when you build the project.

Reply:
[docs | non-blocking]

I do really like that they are not technically necessary to build or contribute, but just make it easier 😊. There might be a small mismatch in the wording, but I don't have a _great_ suggestion for this... One I was thinking was that this might make more sense, in the _Contributing_ section... something along the lines of:


> Secant Wallet uses [SwiftLint](https://github.com/realm/SwiftLint) and [SwiftGen](https://github.com/SwiftGen/SwiftGen) to conform to our coding guidelines for source code and generate accessors for assets. Please install these locally when contributing to the project, they are run automatically on build...

_But_ I don't know if this really makes it any more approachable than what is currently here 😓.

Reply:
updated 🙌

Reply:
excellent!